pkg/signup/service/signup_service.go

```diff
@@ -56,6 +57,13 @@ func NewSignupService(context servicecontext.ServiceContext, cfg ServiceConfigur
 // newUserSignup generates a new UserSignup resource with the specified username and userID.
 // This resource then can be used to create a new UserSignup in the host cluster or to update the existing one.
 func (s *ServiceImpl) newUserSignup(ctx *gin.Context) (*v1alpha1.UserSignup, error) {
+	username := ctx.GetString(context.UsernameKey)
+
+	usersignup.TransformUsername(username)
+	if strings.HasSuffix(username, "crtadmin") {
+		return nil, errors3.NewForbiddenError(fmt.Sprintf("failed to create usersignup for %s", username), "cannot create usersignup for crtadmin")
```

Comment:
could you plaese fix the name of the imported package `errors3` to something more meaningful?

Reply:
@MatousJobanek sure, in another place we use errs, so i updated it to be errs: 6504142457aa5ae8fe870bdc12895a95b6cb0732